### ISSUE TITLE
Some absolute path are hardcoded

### ISSUE THREAD

**Issue by ycollet** [momentarylapse/tsunami#5]
Some absolute pathes are hardcoded in some code. So, if you set and installation path via --prefix the program doesn't work:
- I found some /usr/local in the static/michisoft-tsunami.desktop
- I found some /usr/local/share in src/lib/hui/Application.cpp

**Comment by momentarylapse** [comment]
Yes, that restriction is due to my cheap build system. And my 2 users have not complained so far...
I've been very busy recently. Hopefully next week I can have a look. It might be an easy fix, but I don't want to promise.
For now, if you want to try the program, you can run it without installing. Just `make` it and run `./tsunami` in the repository root folder.
Thanks for your curiosity :)

**Comment by momentarylapse** [comment]
Well, it took a bit longer, due to life. But I finally managed to upgrade the build system.

Now, a `CMakeLists.txt` is generated automatically, and should allow regular `cmake` usage including `--prefix` (at least it worked on my machine).

The `src/lib/hui/Application.cpp` will now respect the prefix via macros (even when using the clumsily provided `Makefile`s that are now mostly obsolete). The `.desktop` file will be changed when installing.

I would very much appreciate it, if you could try it out and complain if it fails. Thanks!

**Comment by ycollet** [comment]
Thanks a lot !
